e2etests/web/regular_integration_tests/lib/screenshot_support.dart

```diff
@@ -20,7 +20,9 @@ import 'package:image/image.dart';
 /// this rate is set to 0.28), since during the end to end tests there are
 /// more components on the screen which are not related to the functionality
 /// under test ex: a blinking cursor.
-const double kMaxDiffRateFailure = 0.5 / 100; // 0.5%
```

Comment:
can we limit this to canvaskit only ?

Reply:
thanks. I'll also discuss this as an option, tomorrow with @yjbanov. As we discussed offline the options we have:
- increasing the diff
- applying different diff for canvaskit
- merging different screenshot files for canvaskit
- (long term, after migration) uploading with different parameters to gold.

Since tests are returning unreliable results now, I'd like to keep this till tomorrow.

Reply:
My guess is, because the tests have been silently failing, as we were upgrading CanvasKit to new versions and weren't updating the screenshots, they slowly deviated from the goldens. Updating the screenshots might do the trick.

Reply:
Could un-updated goldens be causing this LWE build failure?

https://ci.chromium.org/p/flutter/builders/try/Linux%20Web%20Engine/11833?

https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8863763201792734000/+/steps/felt_test_chrome/0/stdout